django/db/models/lookups.py

```diff
@@ -293,6 +316,11 @@ def as_postgresql(self, compiler, connection):
 class Exact(FieldGetDbPrepValueMixin, BuiltinLookup):
     lookup_name = 'exact'
 
+    def __init__(self, *args, **kwargs):
+        from django.db.models import BooleanField
+        self.output_field = BooleanField()
```

Comment:
Are these local imports really needed? Looks like this module imports `django.db.models.fields` just fine?